css/custom.css

```diff
@@ -250,10 +248,6 @@ div.ecc-tab-menu div.list-group > a.active:after {
     border-right: 10px solid transparent;
 }
 
-div.ecc-tab-content {
```

Comment:
Removing this dynamically resizes the content container for the ecc-tab-content which is ok for when there is more content than the 75vh height, however if there is less content such as the media tab - it makes the bottom of the page jump up and goes against the looks we were looking for. 

I think changing this to be a min-height instead of removing it entirely is an acceptable solution, but I cant speak to the responsiveness for mobile or anything like that. 

Let me know what you think @GregoryArmstrong  or if I can elaborate any more. 

Sorry for the delay. Life has been all over the place